src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationConstants.cs

```diff
@@ -13,147 +13,38 @@ public static class LinkedInAuthenticationConstants
     {
         public static class Claims
         {
-            public const string CurrentShare = "urn:linkedin:currentshare";
-            public const string FormattedPhoneticName = "urn:linkedin:phoneticname";
-            public const string Headline = "urn:linkedin:headline";
-            public const string Industry = "urn:linkedin:industry";
-            public const string Location = "urn:linkedin:location";
-            public const string MaidenName = "urn:linkedin:maidenname";
-            public const string NumConnections = "urn:linkedin:numconnections";
-            public const string NumConnectionsCapped = "urn:linkedin:numconnectionscapped";
-            public const string PhoneticFirstName = "urn:linkedin:phoneticfirstname";
-            public const string PhoneticLastName = "urn:linkedin:phoneticlastname";
             public const string PictureUrl = "urn:linkedin:pictureurl";
             public const string PictureUrls = "urn:linkedin:pictureurls";
-            public const string Positions = "urn:linkedin:positions";
-            public const string ProfileUrl = "urn:linkedin:profile";
-            public const string Specialties = "urn:linkedin:specialties";
-            public const string Summary = "urn:linkedin:summary";
         }
 
-        public const string EmailAddressField = "email-address";
+        public const string EmailAddressField = "emailAddress";
 
-        // https://developer.linkedin.com/docs/fields/basic-profile
+        // https://docs.microsoft.com/en-us/linkedin/shared/references/v2/profile/lite-profile?context=linkedin/consumer/context
         public static class ProfileFields
         {
             /// <summary>
-            /// A unique identifying value for the member.
-            ///This value is linked to your specific application.Any attempts to use it with a different application will result in a "404 - Invalid member id" error.
+            /// The unique identifier for the given member. May also be referenced as the personId within a Person URN (urn:li:person:{personId}).
```

Comment:
Can you wrap the _code-y_ indentifiers in `<c>` tags please?

Reply:
Done